### ISSUE TITLE
VoiceOver on mac does not read new lines inside of Visual Studio Code editor

### ISSUE THREAD

**Issue by adincebic** [microsoft/vscode#70033]
- VSCode Version: 1.32.1
- OS Version: macOS Mojave 10.14.3

Steps to Reproduce:

1. Turn VoiceOver on by pressing: command F5.
2. Launch Visual Studio Code
3. Open a file which contains some code or write a method or two.
4. Make sure to interact with the text area by pressing: ctrl option shift down arrow.
5. Now, navigate through the text with up or down arrow and listen what VoiceOver reads.

Expected result:
When the voiceOver encounters a new line, it should read "new line"

Actual result:
When voiceOver encounters a new line, it does not read "new line" it actually repeats the last line that was not empty. Also, it often starts to read the whole content of the file.

To get a better feeling of how it should behave, open a TextEdit or Xcode and repeat the steps above. In TextEdit or Xcode, the VoiceOver reads "new line" when ever it comes across a new line.

Does this issue occur when all extensions are disabled?: Yes/No
Yes

**Comment by adincebic** [comment]
@alexandrudima thanks for your response. I have just reported this issue on the Chromium bug tracker: https://bugs.chromium.org/p/chromium/issues/detail?id=951777

I hope it gets resolved quickly.

**Comment by tayarndt** [comment]
Hi all, Have you found a workaround. This issue is anoying, and i like vs code other than this issue.

**Comment by christopherpross** [comment]
Hi, it is very annoying  that there no updates in the issue on the chromium site. I need viscose for my study and this bug is simply a pain. Maybe we can motivate the developers on chromium to work fodder on this problem?
the best,
Christopher

**Comment by isidorn** [comment]
Just a friendly ping to check if this is still an issue? I just tried latest vscode insiders with latest Mac Catalina and I could not reproduce this. Empty lines are being read out as the new line character.

@marcozehe since I now you use VoiceOver sometimes

**Comment by MarcoZehe** [comment]
I just tried, and indeed when arrowing over a blank line, VoiceOver says „New line“ as in other editors.

**Comment by christopherpross** [comment]
Yeah, I can confirm, It work now. It is not perfect, because voiceover reads the whole text if you hit the end of the texterea, but this is not very important. So this can be closed here.

**Comment by isidorn** [comment]
This is great. Thanks a lot for letting me know.
Yes I also see this issue that VoiceOver reads the whole text if you hit the ned of the textarea. For that I have just fillled this Chromium bug https://bugs.chromium.org/p/chromium/issues/detail?id=1072323
Since I can repro it just with Chrome

Since this issue is fixed I am closing it
